README.md

```diff
@@ -63,7 +64,17 @@ How would I go about accessing the string "pizza" in the above array?
 Describe the rules of scope in JavaScript.
 
 Your Answer:
-```text
+
+"Scope is the parts of a variable / function that can be seen by other functions / variables.  It's kind of difficult to explain, so i'll write an example"
+
+var w = function() {
+var x = "some string";
+someFunction() {
+  var y = "some other string";
+  console.log(var y + var x)
+  }
+}
+var w can "see" all of the functions and variables below it, because they are all nested within it. var x cannot see var y because var y is within someFunction(), but it can see someFunction(), and if i were to return someFunction(), it would be able to see its output. conversly, var y CAN see and utilize the value of var x.
 ```
```

Comment:
Very close. Only thing is that the function stored in `var w` **cannot** see `var y` because the latter is in the scope of `someFunction`. If we were to reference `y` inside of `w`, we would need to return it from `someFunction`.